robo.drupal.example.yml

```diff
@@ -5,6 +5,8 @@ custom_code_paths:
   - 'web/modules/custom'
   - 'web/themes/custom'
   - 'web/sites/*/themes'
+drupal_status_report_ignore_checks:
```

Comment:
Can we add something to the README that explains what these values should be and how to find them?